README.md

```diff
@@ -10,7 +10,8 @@ and use it 3 times inside the `App` instead of static markup.
 - keep the same classNames in `Person.jsx` as in `App.jsx` (`Person`, `Person__name`, `Person__age`, `Person__partner`).
 
 ## Instructions
+
 - Install Prettier Extention and use this [VSCode settings](https://mate-academy.github.io/fe-program/tools/vscode/settings.json) to enable format on save.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_person/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://beliyandr.github.io/react_person/) and add it to the PR description.
```

Comment:
Add the demo link to the PR description
<img width="612" alt="image" src="https://github.com/mate-academy/react_person/assets/64750958/cd15c5e1-13c1-4633-9aa4-e4cc5c4e658a">